src/udev/memory_id/dmidecode.c

```diff
@@ -0,0 +1,768 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```

Comment:
i wonder if the helper should be called dmi_memory_id or so? memory_id sounds overly generic I think? dunno?

Reply:
Renamed to dmi_memory_id.